.travis.yml

```diff
@@ -13,49 +13,62 @@ env:
     - BOOST_ROOT=$HOME/boost_1_60_0
     - BOOST_URL='http://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.gz?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fboost%2Ffiles%2Fboost%2F1.60.0%2Fboost_1_60_0.tar.gz&ts=1460417589&use_mirror=netix'
 
-packages: &gcc5_pkgs
-  - gcc-5
-  - g++-5
-  - python-software-properties
-  - protobuf-compiler
-  - libprotobuf-dev
-  - libssl-dev
-  - libstdc++6
-  - binutils-gold
-  # Provides a backtrace if the unittests crash
-  - gdb
+addons:
+  apt:
+    sources: ['ubuntu-toolchain-r-test']
+    packages:
+      - gcc-5
+      - g++-5
+      - python-software-properties
+      - protobuf-compiler
+      - libprotobuf-dev
+      - libssl-dev
+      - libstdc++6
+      - binutils-gold
+      # Provides a backtrace if the unittests crash
+      - gdb
 
 matrix:
   include:
+    # Default BUILD is "scons".
+
     - compiler: gcc
       env: GCC_VER=5 TARGET=debug.nounity
-      addons: &ao_gcc5
-        apt:
-          sources: ['ubuntu-toolchain-r-test']
-          packages: *gcc5_pkgs
 
     - compiler: gcc
       env: GCC_VER=5 TARGET=coverage
-      addons: *ao_gcc5
 
     - compiler: clang
       env: GCC_VER=5 TARGET=debug CLANG_VER=3.8 PATH=$PWD/llvm-$LLVM_VERSION/bin:$PATH
-      addons: *ao_gcc5
 
     - compiler: clang
       env: GCC_VER=5 TARGET=debug.nounity CLANG_VER=3.8 PATH=$PWD/llvm-$LLVM_VERSION/bin:$PATH
-      addons: *ao_gcc5
+
+    # The gcc cmake build works, but is largely redundant and isn't necessary at this time.
```

Comment:
To clarify, there's no trace of ripple-libpp in the rippled repository. The extra builds we're talking about here are using cmake to build rippled. As we discussed, I'll change one of the existing builds to use cmake instead of adding more. ...I'm thinking gcc.debug.nounity.